devfiles/dotnet/meta.yaml

```diff
@@ -3,3 +3,4 @@ displayName: ".NET Core with Theia IDE"
 description: Default stack with .Net 2.2.105 and Theia IDE
 tags: ["Theia", "Debian", "Dotnet", "C#"]
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+requiredMemory: 2Gi
```

Comment:
I was expecting something more accurate for the memory in the meta.yaml:

```
che-theia:......................................512MB (spec in the plugin meta.yaml)
che-machine-exec-plugin:........................150MB (the default for plugins)
redhat-developer/che-omnisharp-plugin/0.0.1:...1024MB (spec in the devfile)
redhat-developer/netcoredbg-theia-plugin/0.0.1:.512MB (spec in the devfile)
dockerimage dotnet:.............................512MB (spec in the devfile)
--------------------------------------------------------------
TOTAL                                           2.7GB
```

And probably a better naming would be `globalMemoryLimit` (and I am the one to blame for the initial bad naming)